diff-library.go

```diff
@@ -22,14 +21,14 @@ func DiffLibrary() diff.Changelog {
 	return changelog
 }
 
-func main() {
```

Comment:
Ignore, just cleaning up (one main package for all these files so I have to shift main each time)